chunky/src/java/se/llbit/chunky/renderer/projection/OmniDirectionalStereoProjector.java

```diff
@@ -23,7 +23,9 @@
 
 /**
  * A projector for Omni-Directional Stereo (ODS) images. x is mapped to yaw, y is mapped to
- * pitch.
+ * pitch. This projector is like the {@link PanoramicProjector} but can create distinct
+ * images for the left and the right eye to create panoramic stereo images that are perfect
+ * for viewing on VR devices.
```

Comment:
The interesting difference between ODS and equirectangular projection is that they don't produce identical images because the ODS projector slightly displaces the view ray based on the viewing angle to account for the inter-pupillary distance. Equirectangular projection, on the other hand, works more like a pinhole projector - it uses a single point of origin for all rays. So the important difference is that they are not identical and the ray origin is altered in ODS projection.